From: Egbert Eich <eich@suse.de>
Date: Fri Mar 11 13:29:43 2016 +0100
Subject: [PATCH]Define GLAPIVAR separate from GLAPI
Patch-mainline: never
Git-commit: 88dda160564c8c23cb533b0eb258404f1687a2e8

References: bsc#
Signed-off-by: Egbert Eich <eich@suse.com>

GLAPI doesn't contain 'extern' in some environments.
This is needed for variables (GLw).

Signed-off-by: Egbert Eich <eich@suse.de>
---
 Mesa-10.0.3/include/GL/gl.h | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/include/GL/gl.h b/Mesa-10.0.3/include/GL/gl.h
index b484b96..5189008 100644
--- a/include/GL/gl.h
+++ b/include/GL/gl.h
@@ -59,6 +59,7 @@
 #elif (defined(__GNUC__) && __GNUC__ >= 4) || (defined(__SUNPRO_C) && (__SUNPRO_C >= 0x590))
 #  define GLAPI __attribute__((visibility("default")))
 #  define GLAPIENTRY
+#  define GLAPIVAR extern GLAPI
 #endif /* WIN32 && !CYGWIN */
 
 /*
@@ -80,6 +81,10 @@
 #define GLAPI extern
 #endif
 
+#ifndef GLAPIVAR
+#define GLAPIVAR extern
+#endif
+
 #ifndef GLAPIENTRY
 #define GLAPIENTRY
 #endif
